app.js

```diff
@@ -42,3 +45,20 @@ function hideCartHandler() {
   cart.style.transform = "translateX(100%)";
   cartOverlay.style.visibility = "hidden";
 }
+
+for (let i = 0; i < allChevronUpBtns.length; i++)
```

Comment:
you can also shorten the code by writing like this:
allChevronUpBtns.forEach((btn) => { btn.addEventListener...})

Also, the reason to use forEach over index based for loop is that you actually don't need index so why event use it?

Also the same for all down btns.